fuse_core/include/fuse_core/callback_wrapper.h

```diff
@@ -127,13 +141,59 @@ class CallbackWrapper : public ros::CallbackInterface
 // Specialization to handle 'void' return types
 // Specifically, promise_.set_value(callback_()) does not work if callback_() returns void.
 template <>
-inline ros::CallbackInterface::CallResult CallbackWrapper<void>::call()
+inline void CallbackWrapper<void>::call()
 {
   callback_();
   promise_.set_value();
-  return Success;
 }
 
+
+class CallbackAdapter : public rclcpp::Waitable
+{
+public:
+
+  CallbackAdapter(std::shared_ptr<rclcpp::Context> context_ptr);
+
+  /**
+   * @brief tell the CallbackGroup how many guard conditions are ready in this waitable
+   */
+  size_t get_number_of_ready_guard_conditions();
```

Comment:
nit, `override` here and the methods below.

Reply:
https://github.com/locusrobotics/fuse/pull/284/commits/fb5254343a1098ead1608a3d65580615e611f1d6